boards/arm/nucleo_f411re/Kconfig.board

```diff
@@ -7,4 +7,5 @@
 
 config BOARD_NUCLEO_F411RE
 	bool "NUCLEO-64 F411RE Development Board"
+	select I2C_1 if I2C
```

Comment:
Please remove this line

Reply:
This is the same way it is done on the nucleo_f401re and the 96board_carbon. Shouldn't it be standardised in some way?

Reply:
Concerning the carbon board, it makes sense since the i2c bus is fixed on the board, for nucleo board, the pin selection is dependent on the project you make. So it’s place should be in the defconfig so you can change in the app config without any conflicts whatsoever

Reply:
Please take comment into account

Reply:
@superna9999: I had a new though about it and I think we can consider providing default pinmuxing for Arduino connectors on Nucleo boards. It would help quick testing with various Arduino compatible shields.
Thought, it would belong to a global new PR taking into account all STM32 boards. So, I agree we need to remove line right now on this PR (to avoid too much divergences amongst boards today)